src/components/AddComments.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { postArticleVote } from "../utils/api";
+
+export default function AddComments({ article_id }) {
+  const [isPosted, setIsPost] = useState(false);
+  const [newComment, setNewComment] = useState({
+    username: "",
+    body: "",
+  });
+
+  useEffect(() => {
+    setIsPost(false);
+    if (newComment.username !== "") {
+      postArticleVote(article_id, newComment)
+        .then((res) => alert("Posted successfully. Please refresh the page."))
+        .catch((err) => {
+          alert("Please enter a correct username or valid comments. Please refresh the page.");
+        });
+      setIsPost(true);
+    }
+  }, [newComment, article_id]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let obj = {
+      username: e.target[0].value,
+      body: e.target[1].value,
+    };
+    setNewComment((current) => ({ ...current, ...obj }));
+  };
+
```

Comment:
The info that has been typed into your comment box is remaining after you press the submit button.
Remember than text input boxes are [controlled components](https://notes.northcoders.com/courses/js-front-end/controlled-components), and will need a state for the text being input into the form element, which will be transferred over to your newComment state on submitting.